web/src/components/storage/ProposalSettingsSection.jsx

```diff
@@ -112,6 +112,65 @@ const EncryptionSettingsForm = ({
   );
 };
 
+/**
+ * Allows to define snapshots enablement
+ * @component
+ *
+ * @param {object} props
+ * @param {ProposalSettings} props.settings - Settings used for calculating a proposal.
+ * @param {onChangeFn} [props.onChange=noop] - On change callback
+ *
+ * @callback onChangeFn
+ * @param {object} settings
+ */
+const SnapshotsField = ({
+  settings,
+  onChange = noop
+}) => {
+  const rootVolume = (settings.volumes || []).find((i) => i.mountPath === "/");
+
+  const initialChecked = rootVolume !== undefined && rootVolume.fsType === "Btrfs" && rootVolume.snapshots;
+  const [isChecked, setIsChecked] = useState(initialChecked);
+
+  // no root volume is probably some error or still loading
+  if (rootVolume === undefined) {
+    return <Skeleton width="25%" />;
+  }
+
+  const switchState = (_, checked) => {
+    setIsChecked(checked);
+    onChange({ value: checked, settings });
+  };
+
+  const configurableSnapshots = rootVolume.outline.snapshotsConfigurable;
+  const forcedSnapshots = !configurableSnapshots && rootVolume.fsType === "Btrfs" && rootVolume.snapshots;
+
+  const SnapshotsToggle = () => {
+    const explanation = _("Uses btrfs for the root file system allowing to boot to a previous version of the system after configuration changes or software upgrades.");
```

Comment:
Here I would change two things:

* btrfs -> Btrfs
* Use the `<abbr>` wrapping `Btrfs`. you can see examples at https://github.com/openSUSE/agama/blob/76c930c06e1ea4bb24fa11c023cd9a94e09337fd/web/src/components/storage/ProposalDeviceSection.jsx#L409-L418 and https://github.com/openSUSE/agama/blob/76c930c06e1ea4bb24fa11c023cd9a94e09337fd/web/src/components/storage/ProposalSettingsSection.jsx#L80-L89

Reply:
yeap, having in translatable string html tags is definitively something nice....changing name will happen in few minutes